src/components/tabs/themes/light/tabs.base.scss

```diff
@@ -61,6 +64,10 @@ $tabs-animation-function: cubic-bezier(.35, 0, .25, 1);
     height: 100%;
 }
 
+[part='content'] {
```

Comment:
I think the counter part of this is setting `part='headers'` to `flex: none` = 0 0 auto;

Reply:
What if I want bigger headers? i.e. put header content that uses more than 42px?

IMO this is the most used flex rule, you want one fixed part and one flexible part, you use `flex: none` and `flex: 1` works like a charm 99% of the time.

Reply:
Correction, flex: none / flex: 1 -> when you only want to fill the available space.
In this case we want content size to be considered to, so it's flex: none / flex: 1 1 auto, which I believe it's equivalent to the flex-grow: 1 that you have here.